### PR TITLE
Document accessing grafana dashboards

### PR DESCRIPTION
**Pull request by markmark206**
Adding a brief description on how to access Grafana dashboards that come with "Batteries Included" installations.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -271,6 +271,32 @@ Forwarding from [::1]:7233 -> 7233
 and, from a separate window, use the local port to access the service.
 
 
+### Exploring Metrics via Grafana
+
+By default, the full "Batteris Included" configuration comes with a few Grafana dashboards.
```
Comment (shawnhathaway) [inline README.md:276]: `Batteries`

File: README.md [hunk 2]
```diff
@@ -271,6 +271,32 @@ Forwarding from [::1]:7233 -> 7233
...
+
+To access those dashboards, follow the following steps:
+
+1. Exract Grafana's `admin` password from your installation:
```
Comment (shawnhathaway) [inline README.md:280]: `Extract`

File: README.md (hunk 1 above)
Comment (markmark206) [reply README.md]: fixed 🤦‍♂️

File: README.md (hunk 2 above)
Comment (markmark206) [reply README.md]: fixed 🤦‍♂️🤦‍♂️

**Pull request merged**